rooms.js

```diff
@@ -1446,6 +1446,16 @@ let ChatRoom = (function () {
 			this.userList = this.getUserList();
 			this.reportJoinsQueue = [];
 		}
+
+		this.deleteInactive = setTimeout(function () {
+			if (this.active && !this.protect && !this.isOfficial && !this.isPrivate && !this.isPersonal && !this.isStaff) {
```

Comment:
Have to either say when the room is active, or check ``this.messageCount`` to see if a room is actually active or not.